### PR TITLE
Get votes on bills

### PR DESCRIPTION
**Pull request by AriHrannar**
## Status
**Ready**

## Description
Scraping votes for each bill and saving it to the database. Created endpoints to fetch the data as well

### CODE REVIEW

File: src/votes/models.py [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+    '''
+    yes = models.IntegerField(null=True, blank=True)
+    '''
+    Number of people who voted no
+    '''
+    no = models.IntegerField(null=True, blank=True)
+    '''
+    Number of people who did not vote
+    '''
+    didNotVote = models.IntegerField(null=True, blank=True)
```
Comment (Ragnar-H) [inline src/votes/models.py]: I guess the _more_ correct word is 'abstain'

File: src/votes/services.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    return vote_record_soup['atkvæðagreiðslunúmer']
+
+def get_vote_overview(vote_record_soup):
+    return vote_record_soup.find('samantekt')
+
+def get_number_of_votes(kind, vote_overview_soup):
+    return vote_overview_soup.find(kind).find("fjöldi").string
+
+def get_althingi_result(vote_overview_soup):
+    return vote_overview_soup.find('greiðirekkiatkvæði').string
```
Comment (Ragnar-H) [inline src/votes/services.py]: is this the correct soup for `get_althingi_result` ?
I'd think it was `number of didNotVote`

File: src/manage.py [hunk 3]
```diff
@@ -3,8 +3,6 @@
 import sys
 
 if __name__ == "__main__":
-    if not os.path.exists("src/logs"):
```
Comment (Ragnar-H) [inline src/manage.py]: is it safe to remove this?
What about when running in development mode?

File: src/politech_backend/settings/base.py [hunk 4]
```diff
@@ -166,40 +166,24 @@
...
             'handlers': ['console'],
             'propagate': False,
         },
     }
 }
 
 CRON_CLASSES = [
-    "promises.cron_jobs.ConnectBillsAndPromises",
     "case_gather.cron_jobs.GatherCases",
+    "promises.cron_jobs.ConnectBillsAndPromises",
```
Comment (Ragnar-H) [inline src/politech_backend/settings/base.py]: Important change is important 🙈

File: src/politech_backend/settings/production.py [hunk 5]
```diff
@@ -13,7 +13,7 @@
 }
 
 LOGGING['handlers'].update({'sentry': {
-    'level': 'ERROR', # To capture more than ERROR, change to WARNING, INFO, etc.
+    'level': 'INFO', # To capture more than ERROR, change to WARNING, INFO, etc.
```
Comment (Ragnar-H) [inline src/politech_backend/settings/production.py]: Guess the comment can be removed then?

File: src/case_gather/services.py [hunk 6]
```diff
@@ -64,16 +61,21 @@ def createCaseCreators(case_creator_names, case):
...
+            name=case_creator_name
+        )
 
-        if parliament_member.exists():
-            parliament_member = parliament_member.get()
-            CaseCreator.objects.create(
-                case=case,
-                parliament_member=parliament_member
+        if not parliament_member.exists():
+            parliament_member = ParliamentMember.objects.create(
```
Comment (Ragnar-H) [inline src/case_gather/services.py]: +1

File: src/votes/services.py [hunk 7]
```diff
@@ -33,26 +31,30 @@ def get_votes_by_parliament_session(parliament_session):
...
                     name=parliament_member_name
                 )
-                if parliament_member.exists():
-                    vote_result = get_parliament_member_result_from_vote(parliament_member_vote)
-                    Vote.objects.create(
-                        parliament_member=parliament_member.get(),
-                        althingi_result=vote_result,
-                        vote_record=created_vote_record
+                # Create parliamentMember if he does not exist
+                if not parliament_member.exists():
```
Comment (Ragnar-H) [inline src/votes/services.py]: Are there any more services that we need to account for the "create parliament member if not exists"?

File: src/votes/services.py [hunk 8]
```diff
@@ -28,56 +55,29 @@ def get_votes_by_parliament_session(parliament_session):
...
-                else:
-                    parliament_member = parliament_member.get()
-
-                vote_result = get_parliament_member_result_from_vote(parliament_member_vote)
-                Vote.objects.create(
-                    parliament_member=parliament_member,
-                    althingi_result=vote_result,
-                    vote_record=created_vote_record
-                )
+            save_votes(created_vote_record)
```
Comment (Ragnar-H) [inline src/votes/services.py]: \o/

File: src/manage.py (hunk 3 above)
Comment (AriHrannar) [reply src/manage.py]: Should only log into console now or to sentry, we dont have a logger that handles files anymore in the settings :)

File: src/politech_backend/settings/base.py (hunk 4 above)
Comment (AriHrannar) [reply src/politech_backend/settings/base.py]: It runs in that over so best to have GatherCases first :)

File: src/votes/models.py (hunk 1 above)
Comment (AriHrannar) [reply src/votes/models.py]: mmmmmmmmmmmmmmmmmm I guess!

File: src/votes/services.py (hunk 2 above)
Comment (AriHrannar) [reply src/votes/services.py]: hmm ill look into it

**Pull request merged**